scripts/globals/dark_rider.lua

```diff
@@ -0,0 +1,130 @@
+-----------------------------------
+--  Area: Bhaflau Thickets / Caederva Mire / Wajoam Woodlands / Mount Zhayolm
+--   Mob: Dark Rider
+--
+-- Notes: According to testimonies online:
+-- Dark Rider was only ever witnessed before Einherjar's release.
+-- Instant casts death when engaged.
+-- Was unkillable. (However players reported witnessing 99% HP indicated not untouchable)
+-- Dark Rider's minions appear to spawn in random locations along his current path.
+-- When Dark Rider despawns, the hoofprint will appear somewhere in the path that he took
+-----------------------------------
+xi = xi or {}
+xi.dark_rider = xi.dark_rider or {}
+
+local darkRider =
+{
+    16990423, -- Bhaflau Thickets
+    17101136, -- Caedarve Mire
+    17027459, -- Mount Zhayolm
+    16986421, -- Wajaom Woodlands
+}
+local hoofprintTable =
+{
+    16990593, -- Bhaflau Thickets
+    17101292, -- Caedarva Mire
+    17027511, -- Mount Zhayolm
+    16986601, -- Wajaom Woodlands
+}
+
+xi.dark_rider.isDarkRiderEnabled = function()
+    return xi.settings.main.ENABLE_DARK_RIDER
+end
+
+-- This function is called from Aht_Urhgan_Whitegate/Zone.lua
+-- NOTE: Can potentially handle hoofprint logic here when Dark Rider
+--       is disabled and other means to spawn the hoofprint is requied.
```

Comment:
It may be worthwhile to have somewhere where if return xi.settings.main.ENABLE_DARK_RIDER = 0 then have the hoofprints spawn separately without having DR spawn?

Reply:
This is true! I'll put the retail behavior in this section.

Reply:
Take a look :)